pyproject.toml

```diff
@@ -23,6 +79,8 @@ module = [
     "confluent_kafka.*",
     "msgpack.*",
     "pytest_localserver.*",
-    "opentelemetry.*"
+    "opentelemetry.*",
+    "sentry_relay._lowlevel",
+    "_cffi_backend",
 ]
 ignore_missing_imports = true
```

Comment:
If I delete all of this config nothing changes when I run `pre-commit run -a`, are these not used or am I missing something?